src/engine/Sound.ts

```diff
@@ -227,6 +263,11 @@ module ex.Internal {
       public onerror: (e: any) => void = () => { return; };
 
       public load() {
+         // Exit early if we already have data
+         if (!!this._data) {
```

Comment:
Should we be consistent with the above code and go with `this._data !== null` instead?